modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DESeparatorField.testcase

```diff
@@ -80,6 +80,54 @@ definition {
 			fieldName = "Separator");
 	}
 
+	@description = "This is a test for LRQA-70166. This test verifies that Styles are applied to the Separator"
+	@priority = "4"
+	test CanBeStyled {
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Title");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Separator",
+			fieldName = "Separator");
+
+		var styleValue = "border-top: 10px dashed #9D60EB";
+
+		DataEngine.editStyle(
+			fieldLabel = "Separator",
+			index = "1",
+			styleValue = "${styleValue}");
+
+		WebContentStructures.saveCP();
+
+		NavItem.gotoWebContent();
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Title");
+
+		PortletEntry.inputTitle(title = "Web Content Title");
+
+		Button.clickPublish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		for (var style : list "style-dashed,width-10px") {
+			var _style = StringUtil.extractFirst("${style}","-");
+			var value = StringUtil.extractLast("${style}","-");
```

Comment:
@leticiachiu, please rename
- `_style` to `styleName` (the `_` is usually used to name internal / private variables in Java, in addition it seems not to be a pattern in Poshi code)
- `value` to `styleValue`